parachain/runtime/rococo/src/lib.rs

```diff
@@ -376,7 +376,7 @@ type EnsureRootOrHalfLocalCouncil = EnsureOneOf<
 parameter_types! {
 	pub const LocalCouncilMotionDuration: BlockNumber = 7 * DAYS;
 	pub const LocalCouncilMaxProposals: u32 = 100;
-	pub const LocalCouncilMaxMembers: u32 = 4;
+	pub const LocalCouncilMaxMembers: u32 = 8;
```

Comment:
@vgeddes pallet-collective's benchmarks don't work if this value is <= 6 because of things like https://github.com/paritytech/substrate/blob/7b4d55be81c4f35ef9812dad8d0349f43dcce105/frame/collective/src/benchmarking.rs#L221. Is it a problem to increase this?

Reply:
Should be fine. :+1: